hca_ingest/api/requests_utils.py

```diff
@@ -1,9 +1,10 @@
-from requests import Session
+from datetime import timedelta
 from requests.adapters import HTTPAdapter
 from urllib3.util import retry
+from requests_cache import CachedSession
```

Comment:
Did you choose to explicitly use a `CachedSession` rather than [patching](https://requests-cache.readthedocs.io/en/stable/user_guide/general.html#patching) the `requests` library using  `requests_cache.install_cache('ingest_cache')` in order to have more control on cache entry and eviction, methods, parameters and status codes? Was it because exporter is multi-threaded?

Reply:
Ingest client is "a library that will be imported by other libraries or applications", which the readme advises against using the patching method.
The other changes to explicitly use a cached session and to manually revoke urls from the session came from there.